keyboards/tanuki/keymaps/tucznak/keymap.c

```diff
@@ -0,0 +1,116 @@
+        _______, KC_KP_1, KC_KP_2, KC_KP_3, KC_KP_4, KC_KP_5, KC_KP_6, KC_KP_7, KC_KP_8, KC_KP_9, KC_KP_0, _______,
+        _______, KC_NLCK, _______, _______, KC_PPLS, KC_PMNS, KC_PAST, KC_PSLS, KC_COMM, KC_DOT,  _______, 
+                 MO(_FN), _______, _______, LCA(KC_DEL),      LCA(KC_INS),      _______, _______
+    ),
+	
+	/* Up layer (2)
+	 * national and special characters
+     * ,------------------------------------------------------------------------.
+     * | +1  | ě2  | š3  | č4  | ř5  | ž6  | ý7  | á8  | í9  | é0  |  ´  |  ˇ   |
+     * |-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+------|
+     * |  ;°  |     |     |     |  |  |  (  |  )  |  §  |  !  |  /  |  ú  |     |
+     * `------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----'
+     *   |       |     |     |  \  |  %  |  =  |  ¨  |  '  |  -	 |  _  |      |
+     *   `-------+-----+-----+-----+-----+-----+-----+-----+-----+-----+------'
+     *        |      |     |     |            |           |     |      |
+     *        `--------------------------------------------------------'
+     */
+	 
+    [_UP] = LAYOUT(
+        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,          KC_6,          KC_7,    KC_8,          KC_9,          KC_0,          KC_EQL,  LSFT(KC_EQL),
+        KC_GRV,  _______, _______, _______, LSFT(KC_NUBS), LSFT(KC_RBRC), KC_RBRC, KC_QUOT,       LSFT(KC_QUOT), LSFT(KC_LBRC), KC_LBRC, _______,
+        _______, _______, _______, KC_NUBS, LSFT(KC_MINS), KC_MINS,       KC_BSLS, LSFT(KC_BSLS), KC_SLSH,       LSFT(KC_SLSH), _______,
+                 _______, _______, _______, _______,                      _______,                _______,       _______
+    ),
+	
+	/* Navigation layer (3)
+	 * arrows, nav cluster
+     * ,------------------------------------------------------------------------.
+     * |     |     |     |     |     |Home |PgUp |     |     | Up  | Ins | Del  |
+     * |-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+------|
+     * |      |     |     |     |     | End |PgDn |     |Left |Down |Right|     |
+     * `------+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----'
+     *   |       |Caps |PrtSc|ScrLk|Pause|     |     |     |   	 |     |      |
+     *   `-------+-----+-----+-----+-----+-----+-----+-----+-----+-----+------'
+     *        |      |     |     |            |           |     | Menu |
+     *        `--------------------------------------------------------'
+     */
+	 
+    [_NAV] = LAYOUT(
+        _______, _______, _______, _______, _______, KC_HOME, KC_PGUP, _______, _______, KC_UP,   KC_INS,  KC_DEL, 
+        _______, _______, _______, _______, _______, KC_END,  KC_PGDN, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, 
+        _______, KC_CAPS, KC_PSCR, KC_SLCK, KC_PAUS, _______, _______, _______, _______, _______, _______, 
+                 _______, _______, _______, _______,          _______,          _______, KC_MENU
```

Comment:
If you're looking for the modern Menu key (typically located between the right-hand Windows and Ctrl keys), the correct keycode for that is `KC_APP`.

```suggestion
                 _______, _______, _______, _______,          _______,          _______, KC_APP
```